pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -31,7 +31,7 @@
   libXrandr,
   x11Support ? true,
   waylandSupport ? true,
-  nvidiaSupport ? true,
+  nvidiaSupport ? stdenv.hostPlatform == stdenv.buildPlatform,
```

Comment:
```suggestion
  nvidiaSupport ? lib.meta.availableOn stdenv.hostPlatform libXNVCtrl,
```

Reply:
This breaks

Reply:
It also makes little sense. It was `true` before, as in, supported on all platforms. Blocking it on some platforms (but allowing it for some cross) is not the way, if cross is what breaks it.

Reply:
Maybe both conditions in an `&&` might make sense though